pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "pocketsizefund"
+name = "pocketsizefund-monorepo"
```

Comment:
Is there a reason we need this suffix?

Reply:
I just wanted to avoid possible virtualenv collisions with the library which is called `pocketsizefund` (more appropriately)